### PR TITLE
Fix Linux log publish

### PR DESCRIPTION
**Pull request by jaredpar**

### CODE REVIEW

File: .vsts-dnceng.yml [hunk 1]
```diff
@@ -154,7 +154,7 @@ phases:
 
     - task: PublishBuildArtifacts@1
       inputs:
-        PathtoPublish: '$(Build.SourcesDirectory)/Binaries/$(_configuration)/Logs'
+        PathtoPublish: '$(Build.SourcesDirectory)/Binaries/Debug/Logs'
```
Comment (333fred) [inline .vsts-dnceng.yml]: Why does this fix anything?

Comment (333fred) [reply .vsts-dnceng.yml]: Actually, I guess the better question is: why aren't we testing release for this?

Comment (jaredpar) [reply .vsts-dnceng.yml]: The `$(_configuration)` variable isn't defined for this job. It's hard coded to Debug. Only the jobs which do Release + Debug define this.

Comment (jaredpar) [reply .vsts-dnceng.yml]: @333fred Linux is still new. Focusing on getting Debug up + stable before we expand out to release.

**Pull request merged**